### PR TITLE
Em 1800 enable image configuration ccd

### PR DESCRIPTION
**Pull request by isha2040**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EM-1800

### Change description ###
EM-1800 Changes to introduce Image rendering in CCD orchestrator
               * Updated the relevant classes 
               * Added one new class and 2 new Enums
               * Configuration files with Definetion xlsx file
               * Introduced new Funtional Testcases 
               * Updated yaml files and added a new one

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ X] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/em/orchestrator/automatedbundling/AutomatedCaseUpdater.java [hunk 1]
```diff
@@ -45,17 +45,17 @@ public AutomatedCaseUpdater(ConfigurationLoader configurationLoader,
     public JsonNode updateCase(CcdCallbackDto ccdCallbackDto) {
         String configurationName =
                 ccdCallbackDto.getCaseData().has(CONFIG_FIELD) && !ccdCallbackDto.getCaseData().get(CONFIG_FIELD).asText().equals("null")
-                ? ccdCallbackDto.getCaseData().get(CONFIG_FIELD).asText()
-                : CONFIG_MAP.getOrDefault(ccdCallbackDto.getCaseTypeId(), DEFAULT_CONFIG);
+                        ? ccdCallbackDto.getCaseData().get(CONFIG_FIELD).asText()
```
Comment (tomxelliott) [inline src/main/java/uk/gov/hmcts/reform/em/orchestrator/automatedbundling/AutomatedCaseUpdater.java]: is this formatting required?

File: src/main/java/uk/gov/hmcts/reform/em/orchestrator/automatedbundling/AutomatedCaseUpdater.java [hunk 2]
```diff
@@ -45,17 +45,17 @@ public AutomatedCaseUpdater(ConfigurationLoader configurationLoader,
...
         ArrayNode bundles = ccdCallbackDto
-            .findCaseProperty(ArrayNode.class)
-            .orElseGet(() -> {
-                ArrayNode arrayNode = jsonMapper.createArrayNode();
-                ((ObjectNode)ccdCallbackDto.getCaseData()).set(ccdCallbackDto.getPropertyName().get(), arrayNode);
-                return arrayNode;
-            });
+                .findCaseProperty(ArrayNode.class)
+                .orElseGet(() -> {
+                    ArrayNode arrayNode = jsonMapper.createArrayNode();
```
Comment (tomxelliott) [inline src/main/java/uk/gov/hmcts/reform/em/orchestrator/automatedbundling/AutomatedCaseUpdater.java]: an we not just leave this formatting as it was?

File: src/test/java/uk/gov/hmcts/reform/em/orchestrator/automatedbundling/BundleFactoryTest.java [hunk 3]
```diff
@@ -45,6 +45,7 @@ public void create() throws DocumentSelectorException {
             new ArrayList<>(),
             CcdBundlePaginationStyle.off,
             null,
```
Comment (tomxelliott) [inline src/test/java/uk/gov/hmcts/reform/em/orchestrator/automatedbundling/BundleFactoryTest.java]: Do you want to add a test case where the document image isn't null?

Comment (isha2040) [reply src/test/java/uk/gov/hmcts/reform/em/orchestrator/automatedbundling/BundleFactoryTest.java]: new test case created

File: src/main/resources/bundleconfiguration/example-with-filename.yaml [hunk 4]
```diff
@@ -6,6 +6,12 @@ hasCoversheets: false
 hasFolderCoversheets: true
 paginationStyle: topRight
 pageNumberFormat: numberOfPages
+documentImage:
```
Comment (linusnorton) [inline src/main/resources/bundleconfiguration/example-with-filename.yaml]: I think we should omit it on this test
